internal/mkcgo/mkcgo.go

```diff
@@ -0,0 +1,67 @@
+package mkcgo
+
+import (
+	"slices"
+)
+
+// Source is a collection of type definitions and functions.
+type Source struct {
+	TypeDefs []*TypeDef
+	Enums    []*Enum
+	Funcs    []*Func
+	Files    []string
+	Comments []string // Directives starting with "// #"
```

Comment:
From what I see, it has more than directives--maybe:

```suggestion
	Comments []string // All line comments. Directives in this slice start with "#"
```